README.md

```diff
@@ -1,15 +1,75 @@
-A basic Mozilla Ion Study.
+# Mozilla Ion study template.
+The purpose of this repository is to provide a starting point to build an Ion study.
 
-This is an unprivileged WebExtension.
+## Getting started
 
-To test, either load as a temporary add-on in Firefox (`about:debugging`) or Chrome ("developer mode" in `chrome://extensions`) or use `web-ext`:
+1. Fork this repository.
```

Comment:
Humbly, I think we could also suggest using `npx degit mozilla-ion/ion-basic-study my-new-study` as an alternative to forking. This enables the dev to use this repository as scaffolding and removes any unnecessary git-related stuff that comes with forking.

Reply:
@hamilton I believe that's a good idea! I'd love to take it outside of this PR though, so I filed #15 for it.